clients/fides-js/src/lib/i18n/locales/index.ts

```diff
@@ -7,6 +7,7 @@ import type { Locale, Messages, Language } from "..";
  * NOTE: This process isn't automatic. To add a new static locale, follow these steps:
  * 1) Add the static import of the new ./{locale}/messages.json file
  * 2) Add the locale to the LOCALES object below
+ * 3) Add the locale to the LOCALE_LANGUAGE_MAP object below
```

Comment:
Mega nit-pick, but I see the `LOCALE_LANGUAGE_MAP` but now I'm curious where the `LOCALES` obj line 9 is referring to? Has this been renamed to something else?

Reply:
Good catch! I'm assuming it got renamed to `STATIC_MESSAGES` in this doc. Looks like I also need to add this to the tcf version of this file.